tests/server_test_cases/photo_tests.py

```diff
@@ -100,3 +102,32 @@ def test_upload_broken_photo(self):
         assert not doc.cssselect('img.photo')
         assert 'unrecognized format' in doc.text
 
+    def test_set_thumbnail(self):
+        """Tests that a thumbnail is generated."""
+        photo = model.Photo.create(
+            'haiti', image_data=file('tests/testdata/small_image.png').read())
```

Comment:
Can you close the file, maybe using `with` syntax? I guess it won't close automatically? We should fix other tests too, I guess...

Reply:
Done for all the tests in the file.